processes/ingestReport.py

```diff
@@ -27,28 +27,32 @@ def runProcess(self):
 
     @newrelic.agent.background_task()
     def generateReport(self):
-        ingestDate = date.today() - timedelta(days=1)
+        ingestDate = datetime.today()
+        ingestDateStart = ingestDate.replace(hour=0, minute=0, second=0)
+        ingestDateEnd = ingestDateStart + timedelta(days=1)
```

Comment:
What's the purpose of specifically replacing the hours, minutes, and seconds for the ingestDateStart variable?

Reply:
I'm pretty sure `datetime.today()` populates those fields with the values for when the `datetime` object is created. We need `datetime` for the start and end of each day, and this was the easiest way I could think to do that.

If you know a better way of doing that I'm open to it!

Reply:
Oh okay. That makes sense since the start of each day would naturally be all 0s in the 24 hour time format. I can't think of another way to implement this so it's fine with me.